src/server/network/protocol/protocolgame.cpp

```diff
@@ -6399,25 +6395,15 @@ void ProtocolGame::AddPlayerSkills(NetworkMessage &msg)
 	msg.addByte(0xA1);
 
 	msg.add<uint16_t>(player->getMagicLevel());
-
-	if (player->getOperatingSystem() <= CLIENTOS_NEW_MAC)
-	{
-		msg.add<uint16_t>(player->getBaseMagicLevel());
-	}
-
+	msg.add<uint16_t>(player->getBaseMagicLevel());
 	msg.add<uint16_t>(player->getBaseMagicLevel());
```

Comment:
duplicate code

Reply:
I know it's a bit weird but its not duplicated. Several places on the protocolgame you can see some lines like this one. On this both lines what it means is that first we have raw baseMagicLevel and on the second line we have baseMagicLevel with bonus from loyality system. Since we dont have loyality system on the base, we send it duplicated.

Reply:
Oops, I understood  👍🏼